internal/common/clustervalues.go

```diff
@@ -0,0 +1,56 @@
+package common
+
+import (
+	"github.com/giantswarm/clustertest/pkg/application"
+)
+
+// ClusterValues holds common values for cluster-<provider> charts. These are
+// the provider independent values and are present for all the charts
+type ClusterValues struct {
```

Comment:
We need to temporarily support both old and new cluster app Helm values schema, so here we're adding a struct for the new schema.

Team Turtles will provide a new go module with structs for cluster app Helm values, as those are used in other places as well, not just in `clustertest` and `cluster-test-suites`, so the new go module will be auto-generated and kept up-to-date with the schema changes.

Until we get the new module (upcoming weeks), here we do a temporary solution for the sake of unblocking cluster-aws work.